TheForceEngine/TFE_A11y/accessibility.cpp

```diff
@@ -65,11 +75,98 @@ namespace TFE_A11Y  // a11y is industry slang for accessibility
 	static std::vector<Caption> s_activeCaptions;
 	static std::vector<Caption> s_exampleCaptions;
 
+	// Initialize the Accessibility system. Only call this once on application launch.
 	void init()
 	{
+		assert(s_status == CC_NOT_LOADED);
 		CCMD("showCaption", addCaption, 1, "Display a test caption. Example: showCaption \"Hello, world\"");
+		CVAR_BOOL(s_logSFXNames, "d_logSFXNames", CVFLAG_DO_NOT_SERIALIZE, "If enabled, log the name of each sound effect that plays.");
+
+		// Get all caption file names from the Captions directories; we will use this to populate the
+		// dropdown in the Accessibility settings menu.
+		// First we check the User Documents directory for custom subtitle files added by the user.
+		char docsCaptionsDir[TFE_MAX_PATH];
+		const char* docsDir = TFE_Paths::getPath(PATH_USER_DOCUMENTS);
+		sprintf(docsCaptionsDir, "%sCaptions/", docsDir);
```

Comment:
For better linux compatibility, please add
TFE_Paths::mapSystemPath(docsCaptionsDir);
in here (it's a noop on windows).